login/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en" >
+
+<head>
+  <meta charset="UTF-8">
+  <title>Login - globuzzer</title>
+  
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:100" rel="stylesheet">  
+      <link rel="stylesheet" href="login.css">
+
+  
+</head>
+
+<body>
+
+  <div class="main">
+  <div class="bg">
+  </div>
+  
+    
+      
+  <div class="content">
+    <div class="button-container">
+      <button class="button" onclick="materialClick(event)"><div class="circle animate"></div><span class="sign-in">Get Started</span><div class="loader"></div></button>
+    </div>
+    
+      
+       <div class="icon">
+       <img src="images_login/Mark.png" alt="logo">
+      </div>
+      
+      <div class="inputs">
+      
+     
+      
+        <svg class="login" xmlns="http://www.w3.org/2000/svg" width="44" height="40" viewBox="0 0 44 40"><g stroke="#fff" fill="none" stroke-width="3.538" transform="translate(0 -1012.362)"><ellipse ry="8.09" rx="8.244" cy="1022.221" cx="21.555" stroke-linecap="round"/><path d="M1.858 1046.4c-.79 4.74 3.805 4.11 3.805 4.11H37.88s4.846.936 4.312-3.854c-.533-4.79-6.076-10.937-20.04-11.043-13.964-.106-19.504 6.047-20.294 10.786z"/></g></svg>

```

Comment:
never use width and height to svgs (inline), you are using svgs, because they are scalable , by puting the width and height inline you can not control them in css, take a look at the css-tricks svg article